code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -123,3 +123,11 @@
 /mob/living/silicon/flash_act(intensity = 1, override_blindness_check = 0, affect_silicon = 0, visual = 0, type = /obj/screen/fullscreen/flash/static)
 	if(affect_silicon)
 		return ..()
+
+/mob/living/silicon/adjustToxLoss(amount, update_health, forced)
+	adjustBruteLoss(amount, update_health, forced)
```

Comment:
return adjustBruteLoss(amount, update_health, forced)

also you need to make gettoxloss and similar return the appropriate types